packages/x-components/src/views/design-system/design-system-utilities.vue

```diff
@@ -504,7 +345,41 @@
   import { Component } from 'vue-property-decorator';
 
   @Component
-  export default class DesignSystemUtilities extends Vue {}
+  export default class DesignSystemUtilities extends Vue {
+    protected colors = [
+      'lead',
+      'auxiliary',
+      'neutral-10',
+      'neutral-35',
+      'neutral-70',
+      'neutral-95',
+      'neutral-100',
+      'accent',
+      'enable',
+      'disable'
+    ];
+
+    protected maxSize = 20;
+
+    protected get sizes(): string[] {
+      return Array.from({ length: this.maxSize + 1 }).map((_, index) =>
+        String(index).padStart(2, '0')
+      );
```

Comment:
Just FYI, you can pass the `map` as the second `Array.from` parameter:

```suggestion
      return Array.from({ length: this.maxSize + 1 }, (_, index) => String(index).padStart(2, '0'));
```

Reply:
wonderful! thanks